highway.tex

```diff
@@ -563,6 +563,21 @@ \subsection{Leader Selection and Weight Readjustment}
 
 This all works well if the key block is finalized with a sufficient threshold by the time a new era begins. If that is not the case, however, the network's liveness is in danger. Therefore $t_k$ should be large enough to not only finalize the key block, but also for checkpoints to be exchanged and manually verified between the key block and the start of the era. (See section \ref{sectionCheckpoints}.)
 
+\subsection{Independent Eras}
+
+Several aspects of the protocol are simplified by making different eras of the protocol essentially independent of each other. What this means in practice is that each era will be essentially an independent instantiation of the highway protocol with a fixed set of validators and weights and leader schedule (which are determined by the key block and booking block corresponding to that era). The genesis block of this era will have a state inherited from the end block of the previous era (the first block published in that era after that era's end time that is finalized to a sufficient degree of certainty).
+
+Although this view of the era structure simplifies several things, there are some disadvantages. Firstly, it is not clear how to reward or penalize behavior from the validators of the previous era for behavior that takes place after the creation of the final block. In particular, such rewards or penalties cannot be implemented in the previous era as the final block is the last block of that era whose values affect the next era, and therefore we will need some other mechanism. Another potential disadvantage is the time lost during the transition between eras. If the validators of the new era need to wait for the previous era's end block to be finalized, they may lose time before they are able to start.
+
+\subsubsection{Rewards and Penalties}
+
+In order to reward or penalize validators from the previous era for their tail behavior, we will need to implement new transactions that can be called in the new era that make reference to behavior in the previous era. For example, an accusation transaction provides evidence of an equivocation of a validator in the previous era and when executed has that validator's stake slashed accordingly.
+
+We will also need to have an initialization transaction, which includes evidence of a summit finalizing the end block of the previous era to a sufficient degree of certainty. When then transaction is processed, it causes the state of that end block to be adopted along with potentially rewards for the previous era's validators depending on their contributions to the finalization of the end block.
```

Comment:
Do we need a _transaction_ with such a summit? I was thinking that:
* Either you're one of the validators in era 5 already; then you directly observe how the last block gets finalized, and you produce a finality signature once that block is executed.
* Or you're not: Then you wait for the executed finalized last block of era 5 and a sufficient number/weight of finality signatures for that block, and don't need a summit.

Reply:
The concept of this special 'initialization transaction' was to carry rewards for the validators of the previous eras. It should be done 'automatically', but there is no reason to not call such an automatic action a 'transaction'.